<!DOCTYPE html>
<html lang="en">
    <head>
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <meta http-equiv="content-type" content="text/html; charset=utf-8">

      <!-- Enable responsiveness on mobile devices-->
      <!-- viewport-fit=cover is to support iPhone X rounded corners and notch in landscape-->
      <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1, viewport-fit=cover">

      <title>Introduction to Mendelian Randomization</title>
      <meta name="author" content="James McMurray">
      <meta name="description" content="A technical blog about Rust, Linux and other topics.">
      <!-- CSS -->
      <link rel="stylesheet" href="https://jamesmcm.github.io/octozola.css">
      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Sans:400,400italic,700|Abril+Fatface">
      <link href="//fonts.googleapis.com/css?family=PT+Serif:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">
      <link href="//fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">

      <!-- TODO: Move to config -->
      <link href="/favicon.png" rel="icon">
      
      
        <link rel="alternate" type="application/atom+xml" title="RSS" href="https://jamesmcm.github.io/atom.xml">
      
      

      
      
    </head>

    <body>
      
        <header role="banner">
          <hgroup>
		  <h1><a href="/">James McMurray&#x27;s Blog</a></h1>
		  <h2>Rust, Linux and other curiosities</h2>
          </hgroup>
        </header>
      

<nav role="navigation">

<form action="https://www.google.com/search" method="get">
  <fieldset role="search">
	  <input type="hidden" name="q" value="site:jamesmcm.github.io">
    <input class="search" type="text" name="q" results="0" placeholder="Search" data-_extension-text-contrast="">
  </fieldset>
</form>
  
<!-- TODO: Generate from links -->
<fieldset class="mobile-nav">
	<select onChange="window.location.href=this.value">
		<option value="">Navigate…</option>
		<option value="/">» Home</option>
		<option value="/blog">» Blog</option>
		<option disabled>---</option>
		<option value="/atom.xml">» RSS Feed</option>
		<option value="https://www.linkedin.com/in/james-mcmurray/">» LinkedIn</option>
		<option value="https://github.com/jamesmcm">» Github</option>
	</select>
</fieldset>
<ul class="main-navigation">
  <li><a href="/">Home</a></li>
  <li><a href="/blog">Blog</a></li>
</ul>
<ul class="side-links">
  <li><a href="/atom.xml" rel="subscribe-rss" title="Subscribe via RSS">RSS</a></li>
  <li><a href="https://www.linkedin.com/in/james-mcmurray/">LinkedIn</a></li>
  <li><a href="https://github.com/jamesmcm">Github</a></li>
</ul>
</nav>

<div class="content">

<div class="blog-content-page">
<article>
  <h1 class="post-title">Introduction to Mendelian Randomization</h1>
  <span class="post-date">2014-08-17</span>
  <hr class="within-post">
  <p>Mendelian Randomization is an approach to test for a <strong>causal</strong> effect from <strong>observational</strong> data in the presence of certain <strong>confounding</strong> factors.
It uses the measured variation of genes (of known function) to <strong>bound</strong> the causal effect of a modifiable exposure (environment) on a phenotype (disease). The fundamental idea is that the genotypes are <strong>randomly assigned</strong> (due to recombination in meiosis under certain assumptions), and this allows them to be used as an <strong>instrumental variable</strong>.</p>
<span id="continue-reading"></span>
<p>Here is the Directed Acyclic Graph (<strong>DAG</strong>) for the basic set-up. Here <strong>G</strong> is the observed genotype (i.e. the presence of a SNP), <strong>X</strong> is an environmental exposure, <strong>U</strong> is a (possible) unobserved confounder and <strong>Y</strong> is the phenotype (i.e. disease status). This will be explained in detail later.</p>
<p><img src="https://jamesmcm.github.io/blog/mendelian/simpmr.png" alt="Simple Mendelian Randomization" title="Simple Mendelian Randomization" /></p>
<h2 id="motivation">Motivation</h2>
<p>Mendelian Randomization is a useful technique precisely because it allows for causal inference from <strong>observational data</strong>. <strong>Randomised Control Trials</strong> (RCTs) are the gold standard for causal inference, but is not always ethical or possible to carry out RCTs. For example, we cannot randomly assign a lifetime of heavy smoking (or non-smoking) to groups of individuals. This leads to a need to use observational data, however this requires many <strong>assumptions</strong>.</p>
<h2 id="instrumental-variables-ivs">Instrumental Variables (IVs)</h2>
<p>In the previous <strong>DAG</strong>, <strong>G</strong> is the <strong>instrumental variable</strong> (or instrument), because it affects <strong>Y</strong> only through <strong>X</strong> (that is, exclusively, no other path may exist). Therefore, under certain <strong>assumptions</strong> (to be explained later), if <strong>G</strong> is correlated with <strong>Y</strong> then we can infer the edge <strong>X -&gt; Y</strong> (note that <strong>X</strong> must be correlated with <strong>G</strong>).</p>
<p>Now we will consider an example application of instrumental variables, for a hypothetical investigation of the effect of smoking on lung cancer.</p>
<p><img src="https://jamesmcm.github.io/blog/mendelian/smoking.png" alt="Smoking Policy Example" title="Smoking Policy Example" /></p>
<p>If the <strong>assumptions</strong> of the instrumental variable method are met, then if we observe that an increase in tax leads to a reduction in lung cancer, then one can infer that smoking is a "cause" of lung cancer (though perhaps indirectly, i.e. it's not smoking directly that causes cancer but the build-up of tar in the lungs, etc.).</p>
<h2 id="assumptions-of-the-instrumental-variables-method">Assumptions of the Instrumental Variables method</h2>
<p>Assumptions are necessary in all areas of statistics, however it is important to know what assumptions we are making and whether they hold true.</p>
<h3 id="z-x-must-exist">Z -&gt; X must exist</h3>
<p>We must know <strong>a priori</strong> that the causal direction is <strong>Z -&gt; X</strong> and not <strong>X -&gt; Z</strong>. This is what makes the causal structure unique and <strong>identifiable</strong>. Note this does not mean that <strong>Z</strong> has to be the "true" cause of <strong>X</strong>. For example, if <strong>Z</strong> is a SNP, we can choose a SNP in linkage disequilibrium with <strong>Z</strong>, so long as it is independent of all of the other variables, but still correlated with <strong>X</strong>. Note the more correlated that <strong>Z</strong> is with <strong>X</strong>, the better the power will be (i.e. less data required for significant results), if it is not correlated, it cannot be used as an instrumental variable.</p>
<p><img src="https://jamesmcm.github.io/blog/mendelian/snpld.png" alt="Z to X must exist" title="Z to X must exist" /></p>
<h3 id="z-must-be-independent-of-u">Z must be independent of U</h3>
<p>No factor can affect both the instrument and the effects. For example, there cannot be a factor that causes both higher tobacco taxes and less cancer (e.g. if we were comparing rates in different cancers between countries, national health awareness could be such a factor).</p>
<p><img src="https://jamesmcm.github.io/blog/mendelian/noug.png" alt="Z must be independent of U" title="Z must be independent of U" /></p>
<h3 id="no-z-y">No Z -&gt; Y</h3>
<p><strong>Z</strong> cannot directly affect <strong>Y</strong> (or indirectly, except through <strong>X</strong>). I.e. there cannot exist any other mechanisms through which <strong>Z</strong> affects <strong>Y</strong> (i.e. high tobacco tax increases substance abuse, leading to higher rates of cancer).</p>
<p><img src="https://jamesmcm.github.io/blog/mendelian/nozy.png" alt="No Z to Y" title="No Z to Y" /></p>
<h3 id="faithfulness">Faithfulness</h3>
<p>Faithfulness is the assumption that the true underlying DAG will manifest itself in the <strong>observed data</strong> - that is, that the causal effects will not <strong>cancel out</strong>. This is a reasonable assumption, because the contrary would require very specific parameters.</p>
<p>However, note that if relations are deterministic, the implied conditional independencies of the DAG do not hold true, and the assumption of faithfulness is violated. But we will not concern ourselves with this.</p>
<p>Also note that, in practice, the <strong>sample size</strong> is very important in testing the significance of the correlations/independencies in the data.</p>
<h3 id="summary-of-assumptions">Summary of assumptions</h3>
<p>This DAG summarizes the necessary assumptions for the use of instrumental variables:</p>
<p><img src="https://jamesmcm.github.io/blog/mendelian/fulldagass.png" alt="DAG with assumptions" title="DAG with assumptions" /></p>
<h2 id="mendelian-randomization">Mendelian Randomization</h2>
<p>The method of Mendelian Randomization was first introduced in: <em>Apolipoprotein E isoforms, serum cholesterol, and cancer.</em>, Katan MB (1986).</p>
<p>At the time, epidemiologists were wondering whether low serum cholesterol levels increase the risk of cancer. They were known to be correlated, but it could be that latent tumors caused the lower cholesterol levels (i.e. reverse causation), or that both cancer risk and cholesterol levels were affected by another factor, such as diet (i.e. confounding).</p>
<p>However, Katan noticed that patients with Abetalipoproteinemia (a genetic disease that leads to the inability to absorb cholesterol), did not appear predisposed to cancer, despite the predisposition to lower levels of serum cholesterol.</p>
<p>This led Katan to the idea of finding a large group of individuals genetically predisposed to lower cholesterol levels. These individuals are assumed to be the same with respect to other possible confounders (social class, etc.) and so the presence of the cholesterol-affecting allele can be used as an instrumental variable - this is <strong>Mendelian Randomization</strong>.</p>
<p>The Apolipoprotein E (<strong>ApoE</strong>) gene was known to affect serum cholesterol, with the <strong>ApoE2</strong> variant being associated with lower levels. Many individuals carry ApoE2 variant and so have <strong>lower serum cholesterol levels</strong> from birth.</p>
<p>Since genes are <strong>randomly assigned</strong> during meiosis (due to recombination), ApoE2 carriers should not be different from ApoE carriers in any other way (diet, etc.), so there is no confounding via the genome - note these are <strong>assumptions</strong>. It is this which allows the genotype to be used as an instrumental variable in this way (because it is akin to an intervention in a Randomised Control Trial).</p>
<p>Therefore if low serum cholesterol is really causal for cancer risk, the cancer patients should have more ApoE2 alleles than the controls - if not then the levels would be similar in both groups.</p>
<p>Katan only provided the suggestion, but the method has since been used for many different analyses with some success, such as the link between blood pressure and stroke risk. However, some conclusions have later been disproved by Randomised Control Trials. To understand why, we must consider the <strong>biological assumptions</strong>.</p>
<h2 id="biological-assumptions">Biological Assumptions</h2>
<h3 id="panmixia">Panmixia</h3>
<p>Recall the assumption that the genotype is randomly assigned - this implies <strong>panmixia</strong>. That is, there is <strong>no selective breeding</strong> (so random mating). This implies that all recombination of alleles is possible.</p>
<p>In our DAG, this means that <strong>G</strong> is not influenced by <strong>Y</strong> (or any other variables). However, this assumption is not always entirely accurate, as demonstrated by <strong>Population Stratification</strong></p>
<h3 id="population-stratification">Population Stratification</h3>
<p>Population Stratification is the <strong>systematic difference</strong> in allele frequencies between subpopulations, due to ancestry. For example, physical separation leads to <strong>non-random mating</strong>. This then leads to different <strong>genetic drift</strong> in different subpopulations (i.e. changes in allele frequency over time due to repeated random sampling).</p>
<p>This means that the genotype is <strong>not</strong> randomly assigned when considered across different sub-populations, a good example of this is with the difference in rates of lactose intolerance between Northern Europe and Asia.</p>
<h3 id="canalization">Canalization</h3>
<p>Canalization refers to the variation in the <strong>robustness of phenotypes</strong> to changes in the genotype and environment.</p>
<p>The classic example is Waddington's Drosophilia experiment. In the experiment, Drosophilia pupae were exposed to heat shock (i.e. a rapid increase in temperature). Eventually a Cross-veinless phenotype (no cross-veins in wings) was produced. However, by then selectively breeding the resulting flies for this phenotype, it eventually appears without heat shock.</p>
<p>This led to theory of organisms rolling downhill in to "canals" of the <strong>epigenetic landscape</strong> with development, becoming more robust to variation of the environment. We can think of it like an optimization problem which settles in local minima.</p>
<p>The exact biological/molecular mechanisms of canalization are still unknown. With regards to Mendelian Randomization, it can act as a <strong>confounder</strong> between the genotype, environment and phenotype.</p>
<h3 id="no-pleiotropy">No Pleiotropy</h3>
<p>Pleiotropy is the phenomena whereby one gene can affect many (even seemingly unrelated) phenotypes. Mendelian Randomisation makes the assumption of <strong>no pleiotropy</strong>.</p>
<p>In this case, this means that we assume the genotype is only influencing the phenotype via the considered exposure. I.e. ApoE2 <strong>only</strong> affects serum cholesterol levels, and cannot affect cancer risk by other, unobserved means.</p>
<p>This is a big assumption, and <strong>prior knowledge</strong> is necessary. If possible, using multiple, independent SNPs which act through the same path, can help to alleviate this issue, because, if they are all consistent then it is unlikely that they would all have other pathways causing the same change in phenotype. But note that they must be independent, and so cannot be in Linkage Disequilibrium.</p>
<h3 id="the-real-dag">The real DAG?</h3>
<p>Considering the above points, perhaps the true underlying DAG looks more like this:</p>
<p><img src="https://jamesmcm.github.io/blog/mendelian/fulldagbio.png" alt="Full DAG" title="Full DAG" /></p>
<h2 id="conclusion">Conclusion</h2>
<p><strong>Instrumental variables</strong> are a method to infer causal relations from <strong>observational data</strong>, given certain <strong>assumptions</strong>. This method is applied in Genetic Epidemiology with <strong>Mendelian Randomisation</strong>.</p>
<p>This approach has had some success, but the underlying biology still poses some problems with regards to the necessary assumptions. This leaves us with the following questions:</p>
<ul>
<li>
<p>Can we improve the <strong>robustness</strong> of the inference with more measurements of intermediate phenotypes? (such as gene methylation, RNAseq, proteomics measurements, etc.)?<br />
Some work has been done on this under the name of  multi-step Mendelian Randomisation.</p>
</li>
<li>
<p>Can we improve identification of appropriate <strong>instruments</strong>? (e.g. as whole genome sequencing makes it easier to identify population stratification)</p>
</li>
</ul>

</article>
<hr class="end-post">
<nav class="pagination">
    
        <a class="previous" href="https://jamesmcm.github.io/blog/spresource/">‹ Newer Post</a>
    
    <a class="top" href="#top">Back to top</a>
    
    <a class="next" href="https://jamesmcm.github.io/blog/first/">Older Post ›</a>
    
</nav>
</div>

</div>


<footer role="contentinfo">

<p>
Copyright © 2024 - James McMurray -
  <span class="credit">Powered by <a href="https://www.getzola.org/">Zola</a></span>
</p>

</footer>
</body>

</html>
